src/modules/user-auth/components/LoginForm/useLoginForm.ts

```diff
@@ -26,6 +28,8 @@ export const useLoginForm = () => {
       data.password,
       data.keepLogIn
     ).catch((err) => setSubmitError(err.error_description || err.message));
```

Comment:
Can we add this to `getErrorMessge` as well? Or `geFirebaseErrorMessage`?

Reply:
Let's do this in #66